src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -481,35 +503,34 @@ private void testPanelsCollapseExpand() {
         resultsPage.clickGroupByTeam();
 
         resultsPage.displayByGiverRecipientQuestion();
+        clickAjaxLoadedPanelAndWaitForExpansion("panelHeading-section-0-1", "ajax_auto");
```

Comment:
Seems like this method always take "ajax_auto" or "ajax_submit" as the 2nd parameter. Any way to avoid this repetition?
 Can be a separate PR.